contracts/source/BlockVerifier.sol

```diff
@@ -56,6 +56,7 @@ library BlockVerifier {
 			let timestampPrefixPointer := add(gasUsedPointer, gasUsedLength)
 			let timestampPointer, timestampLength := readDynamic(timestampPrefixPointer)
 
+
```

Comment:
```suggestion
```
This appears to be an accident, since an extra newline here doesn't make sense.  😄